src/nameutil.rs

```diff
@@ -45,6 +44,22 @@ pub fn module_name(name: &str) -> String {
     mangle_keywords(name.to_snake()).into_owned()
 }
 
+pub fn enum_member_name(name: &str) -> String {
+    if name.starts_with(|c: char| c.is_digit(10)) {
+        format!("_{}", name.to_camel())
+    } else {
+        name.to_camel()
+    }
```

Comment:
And maybe error if name is not only containing alpha-num-underscore

Reply:
IMHO this unneeded, these members better ignored manually

Reply:
ok :)